common/src/main/java/com/wynntils/models/players/hades/HadesModel.java

```diff
@@ -201,38 +201,28 @@ public void onTick(TickEvent event) {
             float pY = (float) player.getY();
             float pZ = (float) player.getZ();
 
-            if (lastSentStatus != null
-                    && lastSentStatus.equals(
-                            pX,
-                            pY,
-                            pZ,
-                            Models.Character.getCurrentHealth(),
-                            Models.Character.getMaxHealth(),
-                            Models.Character.getCurrentMana(),
-                            Models.Character.getMaxMana())) {
-                tickCountUntilUpdate = 1;
-                return;
+            PlayerStatus newStatus =
+                    new PlayerStatus(pX, pY, pZ, Models.Character.getHealth(), Models.Character.getMana());
+
+            if (lastSentStatus != null) {
+                if (lastSentStatus.equals(newStatus)) {
```

Comment:
Objects.equals would be btter here (I know I did this..)

Reply:
In what way? I think that records per definition implicitly implements an equals method that is identical to Objects.equals.

Reply:
We can get rid of the nested ifs - null check that way

Reply:
I see. I did that by just swapping the order of the comparands.